components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/store/UserSessionStore.java

```diff
@@ -861,6 +861,35 @@ public void storeFederatedAuthSessionInfo(String sessionContextKey, AuthHistory
         }
     }
 
+    /**
+     * Store session details if not exist of a given session context key to map the session context key with
```

Comment:
Shall we rephrase it. Seems some grammatical issue.

Reply:
Resolved.